doc-deploy-dev/action.yml

```diff
@@ -211,6 +211,144 @@ runs:
 
     # ------------------------------------------------------------------------
 
+    - uses: pyansys/actions/_logging@main
+      with:
+        level: "INFO"
+        message: >
+          Copy the 'version/stable/index.html' to 'version/index.html' file. If
+          no stable release is available, copy the 'dev/index.html' instead.
+          Update all references to point to 'stable/_static' or 'dev/_static'.
+
+    - name: "Use the latest 'index.html' as baseline for the versions page"
+      shell: bash
+      run: |
+        if [[ -f 'version/stable/index.html' ]]; then
+          cp version/stable/index.html version/index.html
+        else
+          cp version/dev/index.html version/index.html
+        fi
+
+    - name: "Update all references to point to 'stable/_static' or 'dev/_static'"
+      shell: bash
+      run: |
+        if [[ -f 'version/stable/index.html' ]]; then
+          sed -i "s|_static|stable/_static|g" version/index.html
+        else
+          sed -i "s|_static|dev/_static|g" version/index.html
+        fi
+
+    - name: "Declare the script to read the 'versions.json' file"
```

Comment:
No need to scrap anything. The repository gets checkout first of executing any logic. Therefore, just a list of desired folders versions and pass it to the desired JS code.

Reply:
Thank you, Jorge from the past.